specs/results/results.spec

```diff
@@ -0,0 +1,41 @@
+PIPELINES-26
+# Results pvc tests
+
+Precondition:
+  * Validate Operator should be installed
+  
+## Test Tekton Results: PIPELINES-26-TC01
+Tags: results, e2e
+Component: Results
+Level: Integration
+Type: Functional
+Importance: Critical
+
+Steps:
+  * Create Results route
+  * Ensure that Tekton Results is ready
+  * Configure Results Api
+  * Create project "results-testing" 
+  * Apply in namespace "results-testing"
+    | S.NO | resource_dir                             |
+    |------|------------------------------------------|
+    | 1    | testdata/results/taskrun.yaml  |
+  * Verify taskrun
+    |S.NO|pipeline_run_name           |status    |
+    |----|----------------------------|----------|
+    |1   |results-task                |successful|
+  * Get "taskrun" annotations
+  * Verify "taskrun" Results records
+  * Verify "taskrun" Results logs 
+  * Apply in namespace "results-testing"
```

Comment:
If it could run in a namespace with generated name, I would prefer to split it into two scenarios.